Update change-log.md (#118)
humanitec/about: docs/change-log.md

This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.

## Product Update March 8th, 2024

> - **New:** Introduced Deployment Pipelines which are multi-step workflows used to automate common processes and interactions around Applications and Environments. You can find more information in updated [developer documentation](https://developer.humanitec.com/integration-and-extensions/humanitec-pipelines/overview/) and the [release announcement](https://humanitec.com/blog/feature-announcement-humanitec-deployment-pipelines) on our blog.

Reviewer: ```suggestion
- **New:** Introduced Deployment Pipelines which allow to configure multi-step workflows and automate common processes and interactions around Applications and Environments. You can find more information in the updated [developer documentation](https://developer.humanitec.com/integration-and-extensions/humanitec-pipelines/overview/) and the [release announcement](https://humanitec.com/blog/feature-announcement-humanitec-deployment-pipelines) on our blog.
```

- **Improved:** Deleting a [Resource Definition](https://developer.humanitec.com/platform-orchestrator/resources/resource-definitions) in UI is now also possible from its details view, not only the list of all resource definitions.

#### Versioned component changes:

> - **[Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/):** A new version `1.1.7` has been released.

Reviewer: ```suggestion
- **[Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/):** A new version `1.1.7` has been made available. This release includes a check on agent startup to validate the ids in the [ORGS list](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/#configuration) as valid Humanitec ids.
```

Reviewer 2: This bit LGTM!

- **[Humanitec Operator](https://developer.humanitec.com/integration-and-extensions/humanitec-operator/installation):** A new version has been published. Chart version `0.1.6` and app version `0.10.2` including cookies secrets storage optimization (including GCP cleanup).

> 

Reviewer 3: @sujaya-sys I think this is what the update is about, looks good.

Reviewer: cool, thanks for checking! @MartaD FYI, looks like this is good to go :-)

- **[Humanitec CLI](https://developer.humanitec.com/platform-orchestrator/cli):** Version [0.14.2](https://github.com/humanitec/cli/releases/tag/v0.14.2) is now available. Changes since last announcement:
  - `0.14.0` Introducing additional commands:
    - `humctl score init` simplifies your next score configuration with a score starter file.
    - `humctl score validate` to validate your score file locally.

>     - `humctl score available-resource-types` to list available resource types available to be used.

Reviewer: ```suggestion
    - `humctl score available-resource-types` to list available resource types to be used.
```

>     - `humctl resources` graph to inspect your resource graph.

Reviewer 4: ```suggestion
    - `humctl resources graph` to inspect your resource graph.
```

  -  `0.14.2` is now installing shell completions on osx by default when using brew for an improved UX.

> - **[Humanitec Terraform Provider](https://registry.terraform.io/providers/humanitec/humanitec/latest/docs):** A new version `v1.0.1` has been released, introducting example of a resource definition with secret_refs and private git.

Reviewer 4: ```suggestion
- **[Humanitec Terraform Provider](https://registry.terraform.io/providers/humanitec/humanitec/latest/docs):** A new version `v1.0.1` has been released, introducing example of a resource definition with secret_refs and private git.
```
